.travis.yml

```diff
@@ -4,7 +4,7 @@ cache:
   - "$TRAVIS_BUILD_DIR/build/GSL/src"
   - "$TRAVIS_BUILD_DIR/build/FFTW/src"
   - "$TRAVIS_BUILD_DIR/build/SWIG/src"
-  - "$TRAVIS_BUILD_DIR/build/CLASS/src"
+  #- "$TRAVIS_BUILD_DIR/build/CLASS/src"
```

Comment:
Why is this commented out? We should be able to reset caches by click around in travis. Also, the caching in travis needs to be version aware.

Reply:
Yes, this was commented out for testing (since caching CLASS 2.6.x would be Very Bad for this PR). I will uncomment.